README.md

```diff
@@ -45,6 +45,30 @@ logger.info("Hello World!")
 
 For further information, please check the [API documentation][api-docs].
 
+### Selecting a logging backend implementation (applications only)
```

Comment:
Imported this section and adjusted from https://github.com/apple/swift-metrics#selecting-a-metrics-backend-implementation-applications-only

Putting more focus on that it's possible to do this and hopefully getting more interest in implementing some backends?